cli/slsa-verifier/main_test.go

```diff
@@ -854,18 +857,41 @@ func Test_runVerifyGCBArtifactImage(t *testing.T) {
 			}
 
 			for _, v := range checkVersions {
+				semver := path.Base(v)
+				builderID := pString(builder + "@" + semver)
 				provenance := filepath.Clean(filepath.Join(TEST_DIR, v, tt.provenance))
 				image := tt.artifact
 				var fn verify.ComputeDigestFn
-				if !tt.oci {
+				// Select the right image according to the builder version we are testing.
+				if strings.Contains(image, `%s`) {
+					image = fmt.Sprintf(image, semver)
+				}
+				// Add the sha256 digest to the image name, if provided.
+				if len(tt.artifactDigest) > 0 {
+					digest, ok := tt.artifactDigest[semver]
+					if !ok {
+						panic(fmt.Sprintf("%s not present in artifactDigest %v", semver, tt.artifactDigest))
+					}
+					image = fmt.Sprintf("%s@sha256:%s", image, digest)
+				}
+				// If it is a local image, change the digest computation.
+				if !tt.remote {
 					image = filepath.Clean(filepath.Join(TEST_DIR, v, image))
 					fn = localDigestComputeFn
 				}
 
+				// If builder ID is set, use it.
+				if tt.pBuilderID != nil {
```

Comment:
small nit: could you move this code above where builderID is defined so it's clear it gets overridden if one is provided? or move that down here

Reply:
done, PTAL